Makefile

```diff
@@ -25,8 +25,15 @@ debug: doc
 .PHONY: test
 test:
 	$(shell mkdir -p test/data;  cd test/data; ../scripts/gen_certs.sh; cd ${TOP_LEVEL}; sudo skopeo --insecure-policy copy -q docker://public.ecr.aws/t0x7q1g8/centos:7 oci:${TOP_LEVEL}/test/data/zot-test:0.0.1;sudo skopeo --insecure-policy copy -q docker://public.ecr.aws/t0x7q1g8/centos:8 oci:${TOP_LEVEL}/test/data/zot-cve-test:0.0.1)
+	$(shell sudo mkdir -p /etc/containers/certs.d/127.0.0.1:8088/; sudo cp test/data/client.* /etc/containers/certs.d/127.0.0.1:8088/; sudo cp test/data/ca.* /etc/containers/certs.d/127.0.0.1:8088/;)
+	$(shell sudo chmod a=rwx /etc/containers/certs.d/127.0.0.1:8088/*.key)
 	go test -tags extended -v -race -cover -coverpkg ./... -coverprofile=coverage.txt -covermode=atomic ./...
 
+.PHONY: test-cleaner
```

Comment:
Make this a part of "clean" target itself, why create a new one

Reply:
I think it should belong to the test side and for this reason I put it separately.